StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1648UnitTests.cs

```diff
@@ -18,6 +17,87 @@ namespace StyleCop.Analyzers.Test.DocumentationRules
     /// </summary>
     public class SA1648UnitTests
```

Comment:
I don't see any tests for when it **should** trigger, for example if there is a constructor in the base class but it has the wrong number of parameters, or if it has the same number of arguments but one of them is of the wrong type.